cpu/native/Kconfig

```diff
@@ -12,6 +12,8 @@ config CPU_ARCH_NATIVE
     select HAS_CPP
     select HAS_CPU_NATIVE
     select HAS_PERIPH_CPUID
+    select HAS_PERIPH_GPIO
```

Comment:
Isn't this only provided when `OS` is Linux? If so, it should be selected by `NATIVE_OS_LINUX` like `HAS_PERIPH_SPI`

Reply:
Ok, fixed.